source/views/components/nav/Navbar.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+  import type { Tab } from "./tab";
+
+  let {
+    Tabs = $bindable([]),
+    current = $bindable(""),
+  }: { Tabs: Tab[]; current: string } = $props();
+</script>
+
+<div class="tab">
+  {#each Tabs as tab}
+    {#if current == tab.label}
+      <button
+        class="nav-button selected"
+        onclick={() => {
+          current = tab.label;
+        }}><span class="text">{tab.title}</span></button
+      >
+    {:else}
+      <button
+        class="nav-button not-selected"
+        onclick={() => {
+          current = tab.label;
+        }}><span class="text">{tab.title}</span></button
+      >
+    {/if}
```

Comment:
instead of duplicate i think svelte can do something like
class:selected={current === tab.label}

Reply:
and then instead of not-selected you can do css :not(.selected) or something i think

Reply:
funky wonky css hvis man gør det. Magter jeg ikke lige